### DISCUSSION TITLE
Test du module zigbee Nodon special volet roulant SIN-4-RS-20 - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#26] [Comments]
# Test du module zigbee Nodon special volet roulant SIN-4-RS-20 - Haade.fr

Nodon produit plusieurs modules zigbee 3.0 de qualité, dans cet article je vais présenter le module de contrôle de volet roulant SIN-4-RS-20

https://haade.fr/fr/blog/test-du-module-zigbee-volet-roulant-nodon-SIN-4-RS-20

**Comment by guilaul** [comment]
Bonjour,
Pour information, avec des volets roulants ayant une motorisation PROFALUX filaire simple, ces modules, comme tous les autres je pense , ne sont pas compatibles.
Le site de Profalux indique que :
"Le module domotique :
• ne doit pas comporter de contrôle de courant (ou sa désactivation doit être possible),
• doit être compatible avec des fins de course électroniques,
• ne doit pas envoyer simultanément un ordre de « montée » et « descente »."

Pour mon cas j'ai utiliser un scénario pour commander 2 fois la commande ( montée par exemple ) avec une pause de 4s entre les 2 actions.
Si une seule action est commandé, aucun mouvement inititié car le mode sécurité du module s'active.
Le support tehcnique de NODON m'a indiqué qu'il n'était pas possible de déactivé le mode : controle du courant

**Comment by nrozron** [comment]
Bonjour, comme indiqué dans la [doc constructeur](https://nodon.fr/wp-content/uploads/2021/04/NodOn_Zigbee_RollerShutterModule_SIN-4-RS-20-Datasheet_FR.pdf), ce module fonctionne parfaitement avec un volet filaire Somfy (et ZigBee2Mqtt).
